suite-native/app/src/App.tsx

```diff
@@ -2,50 +2,63 @@ import React, { useEffect } from 'react';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { Provider, useDispatch } from 'react-redux';
+import { IntlProvider } from 'react-intl';
 
 import { NavigationContainer } from '@react-navigation/native';
 
+import enMessages from '@trezor/suite-data/files/translations/en.json';
```

Comment:
Please add comment here that this is only temporary until Intl refactor will be finished.